backstory/src/main/java/com/google/sps/servlets/data/BlobstoreManagerFactory.java

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets.data;
+
+import com.google.sps.servlets.data.BlobstoreManager;
+
+/**
+ * Factory pattern for object/mock creation.
+ */
+public interface BlobstoreManagerFactory { public BlobstoreManager newInstance(); }
```

Comment:
Would be good to have Javadocs for `newInstance()` method.

Reply:
Added this.